docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -96,6 +115,10 @@
     * When the survival plot is zoomed in and an image is downloaded, the curves within the image may extend beyond the y-axis. <!--SV-2348-->
 * __Gene Expression Clustering__:
     * The tool allows deleting the gene expression group and displays an uninformative error message after submitting the deletion.
+* __Cohort MAF__:
+    * A downloaded file may be corrupted if the server data processing is terminated after 5 minutes in order to conserve server resources.
+    There will be a red banner above the MAF contorls to indicate the termination.
```

Comment:
```suggestion
    * There will be a red banner above the MAF controls to indicate the termination.
```

Reply:
The lack of asterisk before "There will be .." is intentional. There is no extra space in the preceding line/list item, so this new line will be considered part of the previous list entry/bullet item.